### DISCUSSION TITLE
consiglio tapparelle

### DISCUSSION THREAD

**Discussion by SkunkGuru** [indomus/forum#2488] [GENERALE - Discussioni generiche]
salve a tutti,
sto pensando a come motorizzare le tapparelle per gestirle (ovviamente) da HomeAssistant

Ho visto quegli avvolgitori che si installano direttamente al posto della molla; sembrano una alternativa facile all'installazione del motore tubolare nel cassonetto. Potrebbero essere una alternativa valida per le tapparelle più piccole (finestre) ? Ma soprattutto, esistono dei modelli che siano poi pilotabili da HA ? Quelli che ho trovato sembrano non avere possibilità di integrazione

**Comment by federoco** [comment]
Ciao, io ti suggerisco di usare un motore tubolare, in quanto regge meglio gli sforzi ed offre un movimento più uniforme quando alzi e abbassi le tapparelle. I Somfy sono ottimi e domotizzabili, a patto di avere portato all'avvolgibile fase, neutro e messa a terra dedicati.

Comunque, a prescindere dalla soluzione scelta (avvolgitori o motori tubolari), per domotizzarli ti basta poter collegare a monte uno switch come lo Shelly 2.5.
Puoi staccarti dal suo cloud e configurarli in HA tramite MQTT.

**Reply by SkunkGuru** [reply]
pensavo all'avvolgitore perchè mi sembrava più semplice come installazione; per il tubolare ritengo andrà chiamato un professionista.

pensavo già allo shelly, ma bisogna (se non erro) che il comando abbia la doppia funzione alza/abbassa che il tubolare avrà sicuramente per farlo arrivare al pulsante fisico, mentre l'avvolgitore lo integra il pulsante, quindi non credo sarebbe comandabile da remoto

**Reply by federoco** [reply]
ciao, il tubolare puoi installarlo anche da solo, a patto di essere un po' pratico.
L'unica difficoltà che puoi trovare è quella di settare bene il fine corsa. Ogni motore ha il suo sistema. 

Non ho capito il tuo discorso il discorso del pulsante integrato nell'avvolgitore... hai un immagine esplicativa?

**Reply by SkunkGuru** [reply]
non credo di poter mettere link, comunque come esempio copio questo

![image](https://github.com/indomus/forum/assets/15073199/96d5812e-460c-4a7e-9ec5-153490241bb6)

**Comment by cl-7k-01** [comment]
Ti consiglio anche io il motore tubolare. Oltre agli shelly 2.5 ora esiste un nuovo prodotto Shelly Pro Dual Cover, da mettere però in quadro, e il lavoro sarebbe "pulitissimo" (ma molto più costoso...)

**Reply by Marcunz** [reply]
interessa anche a me questa soluzione più pulita nel quadro, come funziona? Devo portare un (?) filo dal quadro ad ogni tapparella?
Sto facendo un nuovo impianto elettrico: questo shelly governa una sola tapparella?
